test/transport.test.js

```diff
@@ -46,15 +46,15 @@ describe('transport', function () {
       var listen = Transport.listen(_.noop)
       var seneca = {
         log: {
-          info: function () {
-
-          }
+          info: _.noop,
```

Comment:
Can we just use a `function noop () {}` in common and remove the _ dependency in this file?